templates/base_index.html

```diff
@@ -101,7 +101,15 @@ <h2 class="p-heading--three u-no-max-width">
     <section class="p-strip is-deep u-image-position is-bordered">
       <div class="row">
         <div class="col-7">
-          <img src="https://assets.ubuntu.com/v1/0bfbd13b-cloud_section_bundle.svg?h=460" class="u-image-position--top u-hide--small" alt="" height="460" style="height: 100%; right: 47%;" />
+          {{
```

Comment:
This is a different image - expected?

Reply:
Yeah, the Juju one was complicated with image position but I asked Spencer for a more simpler image.